### ISSUE TITLE
TreeTable: treeTableToggler missing aria-label

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#14271]
### Describe the bug

WCAG 2.1: Button missing a text alternative. Provide a text alternative for the button
TreeTableToggler.

Add ariaLabel -property to the TreeTableToggler -component.

### Environment

Windows, Chrome

### Reproducer

https://primeng.org/treetable

### Angular version

17.0.0-rc.1

### PrimeNG version

17.0.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.6.0
